object.go

```diff
@@ -657,3 +657,30 @@ func checkSubset(outer, inner map[string]interface{}) bool {
 	}
 	return true
 }
+
+// Every runs the passed function for all the key value pairs in the object
+// value should be map[string]interface{} or struct.
```

Comment:
I guess this line:

```
// value should be map[string]interface{} or struct.
```

should be removed

Reply:
Sure.